modules/identity/src/Volo.Abp.Identity.EntityFrameworkCore/Volo/Abp/Identity/EntityFrameworkCore/EfCoreIdentityUserRepository.cs

```diff
@@ -301,4 +302,13 @@ public override async Task<IQueryable<IdentityUser>> WithDetailsAsync()
     {
         return (await GetQueryableAsync()).IncludeDetails();
     }
+
+    public virtual async Task<IdentityUser> GetUserByTenantIdAndUserNameAsync(Guid tenantId, [NotNull] string userName, bool includeDetails = true, CancellationToken cancellationToken = default)
```

Comment:
* You've declared, but not used the `includeDetails` parameter. See other similar methods to understand how to use.
* If a method may return null, then its name should start with `Find`, not `Get`
* A good naming can be `FindByTenantIdAndUserNameAsync`. Don't use `User`, we are already in the user context.